conf/client.conf

```diff
@@ -152,6 +152,14 @@ global.metricDummyServerStartPort=9000
 # 是否关闭健康检查: true/关闭 false/不关闭
 global.turnOffHealthCheck=true
 
+# minimal open file limit
+# if set value to 0, then will skip check and set open file limit
+# NOTE: open file limit will affect how may sockets we can create,
+#       the number of sockets is related to the number of chunkserver and mds in the cluster,
+#       and during some exception handling processes, client will create additional sockets
+#       the SAFE value is 2 * (#chunkserver + #mds)
+global.minOpenFileLimit=65536
```

Comment:
each client 65536?

Reply:
Lowering it to 1024 and it's fine in most cases.